jekyll/_cci2/workflows.md

```diff
@@ -90,6 +91,36 @@ workflows:
 ```
 
 The dependencies are defined by setting the `requires:` key as shown. The `deploy:` job will not run until the `build` and `test1` and `test2` jobs complete successfully. A job must wait until all upstream jobs in the dependency graph have run. So, the `deploy` job waits for the `test2` job, the `test2` job waits for the `test1` job and the `test1` job waits for the `build` job.
+
+## Holding a Workflow for a Manual Approval
+
+Workflows may be configured to wait for manual approval of a job before continuing by using the `type: approval` key. The `type: approval` key is a special job type that is **only** added under in your `workflow` key. This enables you to configure a job with `type:approval` in the workflow before a set of parallel jobs that must all wait for manual approval. Jobs run in the order defined until the workflow processes a job with the `type: approval` key followed by a job on which it depends as in the following `config.yml` example:
+
+```
+workflows:
+  version: 2
+  build-test-and-approval-deploy:
+    jobs:
+      - build
+      - test1:
+          requires:
+            - build
+      - test2:
+          requires:
+            - test1
+      - test3:
+         type: approval
+         requires:
+           - test1
+           - test2
+      - deploy:
+          requires:
+            - test3
```

Comment:
This is actually not a great example, as it's three Approval jobs chained together. Also, the UI is currently broken because there's a bug in workflows which begin with an Approval job. (Also, beginning with an Approval job is a bit of an odd thing to do in the first place.) Let's rite a more canonical example, preferably in a controlled test repo and not in our frontend repo.